guide/daily-spending-wallet/sending.md

```diff
@@ -175,7 +175,7 @@ If no amount is provided via a payment request, manual entry should be simple an
 
 **Metadata**
 
-A transaction history is hard to make sense of when it only shows amounts, dates, and identifiers. Users should be allowed to add descriptions, tags, and other metadata to add context. This context can separately be used for helpful tools like visual spending breakdowns.
+A [transaction history]({{ '/guide/daily-spending-wallet/activity/' | relative_url }}) is hard to make sense of when it only shows amounts, dates, and identifiers. Users should be allowed to add descriptions, tags, and other metadata to add context. This context can separately be used for helpful tools like visual spending breakdowns.
```

Comment:
```suggestion
[Transaction history]({{ '/guide/daily-spending-wallet/activity/' | relative_url }}) is hard to make sense of when only shown amounts, dates, and identifiers. Users should be allowed to add descriptions, tags, and other metadata to add context. This can be helpful to visualize spending breakdowns.
```

Reply:
First few edits make the sentence grammatically incorrect. Disagree with the other changes.